Sources/Hackle/Core/Event/UserEventProcessor.swift

```diff
@@ -184,11 +188,15 @@ class DefaultUserEventProcessor: UserEventProcessor, AppStateListener {
 
         let pendingCount = eventRepository.countBy(status: .pending)
         if pendingCount >= eventFlushThreshold && pendingCount % eventFlushThreshold == 0 {
-            dispatch(limit: eventFlushMaxBatchSize)
+            flush()
```

Comment:
```suggestion
            flushInternal()
```
이미 queue에서 실행되고 있는 상태라 다시 queue에 넣으면 안됩니다